### ISSUE TITLE
Working directory setting is ignored

### ISSUE THREAD

**Issue by guillep2k** [a85/POSTMan-Chrome-Extension#9510]
**Describe the Issue**

Even when I store my files in the folder set as "working directory" and I attempt to use them as source data for a request, they show up with the warning _"This file isn't in your working directory"_.

**Steps to Reproduce**

With default settings:

- Go to Settings, General, Working Directory, check that the Working Directory is set by default to my user's home directory (`D:\Users\myname\Postman\files`).
- Set up a `PUT` request.
- Select a `binary` body.
- Press the "select file" button.
- Go to my working directory and select the file I want to use.
- The file shows up with the "not in your Working Directory" warning.

With custom settings:

- Go to Settings, General, Working Directory, set my Working Directory to a specific directory in my disk (`C:\Postman`).
- Set up a `PUT` request.
- Select a `binary` body.
- Press the "select file" button.
- Go to my newly selected working directory and select the file I want to use.
- The file shows up with the "not in your Working Directory" warning.

**Note:** Turning ON and OFF the "Allow reading files outside working directory" has no effect.

**Screenshots or Videos**

![image](https://user-images.githubusercontent.com/18600385/106395692-5c771980-63e2-11eb-995c-75a1433bcadb.png)

![image](https://user-images.githubusercontent.com/18600385/106395739-9811e380-63e2-11eb-9d9c-67311b613da0.png)

![image](https://user-images.githubusercontent.com/18600385/106395795-f212a900-63e2-11eb-9371-e37279ee18a3.png)

![image](https://user-images.githubusercontent.com/18600385/106395807-ff2f9800-63e2-11eb-802c-36edee848610.png)

![image](https://user-images.githubusercontent.com/18600385/106395815-05be0f80-63e3-11eb-950a-a734c0ad5950.png)

![image](https://user-images.githubusercontent.com/18600385/106395826-1a020c80-63e3-11eb-921f-00bcdcf527d0.png)

![image](https://user-images.githubusercontent.com/18600385/106395860-4e75c880-63e3-11eb-9155-f5c55e55e297.png)

![image](https://user-images.githubusercontent.com/18600385/106395865-5a618a80-63e3-11eb-951b-e8daaa608cc8.png)

![image](https://user-images.githubusercontent.com/18600385/106395882-6b120080-63e3-11eb-8e8e-4c6e7fe78003.png)

**Environment Information**

 - Platform Type: Desktop App
 - Postman Version: 8.0.3
 - Operating System: Windows 10 Pro version 20H2 build 19042.746

**Comment by danmichaelo** [comment]
I'm also seeing this issue with

- Platform Type: Desktop App
- Postman Version: 8.1.0
- Operating System: Mac OS 10.15.7 (19H524)

**Comment by aeriondyseti** [comment]
Also seeing this issue:

Platform Type: Desktop App
Postman version: 8.3.0
Operating System: Windows 10

**Comment by nidhidesai990** [comment]
Same issue. I am trying to run my newman-cli runner and my file is not getting picked up and my all test-rail cases are failing.

**Comment by guillep2k** [comment]
It seems to me that as of Postman v8.11.1 (probably earlier) the problem is fixed. I'm currently experimenting no such issue. I leave it open in case other user confirms the issue still exists. @nidhidesai990 @AerionDyseti @danmichaelo

**Comment by guillep2k** [comment]
The issue is fixed for me. Since there's no further feedback, I'm closing the issue.

**Comment by michahell** [comment]
This still doesn't work at all on `Windows 10 20H2 19042.1466` when creating a Postman dump, selecting the folder with that postman dump as working directory. I have a feeling this way of working is being frustrated on purpose due to the existence of Postman teams, or no effort is being put into maintaining this way of working.

**Comment by rajeshisnepali** [comment]
Not working in Postman for Linux
Version 9.17.2

**Comment by jeffreybeale** [comment]
This was still a problem for me in Postman for Windows (version 10.14.3) and I figured out how to get Postman to "forget" the last directory from where I had selected a data file (ages ago).  There was a Preference file for me under:
`C:\Users\Your.User.Name\AppData\Roaming\Postman\Partitions\30ce0954-87f6-4cd4-8374-48407708735e`

This Preferences file is in JSON format and it had an element:
`"selectfile":{"last_directory":"C:\\Users\\LONG_PATH_WHERE_POSTMAN_WAS_ALWAYS_DEFAULTING_THE_DATA_FILE_LOAD"}`
I removed this entry from the Preferences file and Postman started to use my configured Working directory Location.

I hope this helps someone else frustrated by this behavior.
